gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterConfigurationKeys.java

```diff
@@ -159,4 +159,7 @@
   public static final String KILL_DUPLICATE_PLANNING_JOB = GOBBLIN_CLUSTER_PREFIX + "kill.duplicate.planningJob";
   public static final boolean DEFAULT_KILL_DUPLICATE_PLANNING_JOB = true;
 
+  public static final String SHOULD_CANCEL_RUNNING_JOB_ON_DELETE = GOBBLIN_CLUSTER_PREFIX + "shouldCancelRunningJobOnDelete";
```

Comment:
suggestion: drop the prefix "should" from this config. So config could be: cancelRunningJobOnDelete

Reply:
+1, same with variable name :) 
btw, should this be `job.cancelR unningOnDelete`, since it's a job property just like `job.schedule` ?

Reply:
Since this config is specific to cluster jobs, I added the GOBBLIN_CLUSTER_PREFIX to the config. Maybe, gobblin.cluster.job.cancelRunningOnDelete?